test.py

```diff
@@ -1,48 +1,50 @@
+from utils import SplittedDataLoader

```

Comment:
it's better to group statements `from smt import SMT` and `import something`. And it's better to start file with import statements.